pkg/api/types.go

```diff
@@ -1993,6 +1993,8 @@ const (
 	NodeOutOfDisk NodeConditionType = "OutOfDisk"
 	// NodeMemoryPressure means the kubelet is under pressure due to insufficient available memory.
 	NodeMemoryPressure NodeConditionType = "MemoryPressure"
+	// NodeNetworkingReady means that network for the node is correctly configured.
+	NodeNetworkingReady NodeConditionType = "NetworkingReady"
```

Comment:
It appears that we are settling on node conditions being expressed with opposite polarity.

see discussion:
https://github.com/kubernetes/kubernetes/issues/25773#issuecomment-221043286

Basically, if a condition has State=True, its considered a problem.

I am not sure if this PR demonstrates that this is a bad decision as it seemed natural to expose it in this fashion.

/cc @vishh